build.sh failed with "Execution Failed: no install script" message

* .NET Core Version: 3.0.103 (Commit:  b7ef045669)
* Platform/OS: ubuntu linux 18.04-x64
* Winforms branch version (date): Thu Mar 26 07:37:13 2020 -0700

**Problem description:**
I cloned the current version of Winforms and tried to build it local, and I got error message (below with the command):
```
maestro@mmeniac:~/.vscode/extensions/WinForms/winforms$ ./build.sh
/home/maestro/.vscode/extensions/WinForms/winforms/eng/common/native/install-dotnet-api-docs_netcoreapp3.0.sh --baseuri https://netcorenativeassets.blob.core.windows.net/resource-packages/external --installpath /home/maestro/.netcoreeng/native//bin --version 0.0.0.1
Execution Failed: no install script
```
Before I updated file global.json with my actual dotnet versions in such way:

_{
  "tools": {
    "dotnet": "3.0.103",
    "runtimes": {
      "dotnet": [
        "$(MicrosoftNETCoreAppPackageVersion)"
      ]
    }
  },
  "sdk": {
    "version": "3.0.103"
  },
  "msbuild-sdks": {
    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20171.1",
    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20171.1",
    "FIX-85B6-MERGE-9C38-CONFLICT": "1.0.0",
    "Microsoft.NET.Sdk.IL": "5.0.0-alpha1.19513.3"
  },
  "native-tools": {
    "dotnet-api-docs_netcoreapp3.0": "0.0.0.1"
  }
}_

Probably I should make some other changes in global.json?
(but there is no explanations in corresponding documentation file about its parameters)

also I got some other errors during the build process:

Build FAILED.
CSC : error CS1566: Error reading resource 'System.Windows.Forms.Design.professional1' -- 'Could not find file '/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/Resources/System/WinForms/Design/professional1.ico'.' [/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj]
    0 Warning(s)
    1 Error(s)
CSC : error CS1566: Error reading resource 'System.Windows.Forms.Design.professional2' -- 'Could not find file '/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/Resources/System/WinForms/Design/professional2.ico'.' [/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj]
    0 Warning(s)
    1 Error(s)
CSC : error CS1566: Error reading resource 'System.Windows.Forms.Design.Behavior.leftopen' -- 'Could not find file '/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/Resources/System/WinForms/Design/Behavior/leftopen.ico'.' [/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj]
    0 Warning(s)
    1 Error(s)
CSC : error CS1566: Error reading resource 'System.Windows.Forms.Design.Behavior.leftclose' -- 'Could not find file '/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/Resources/System/WinForms/Design/Behavior/leftclose.ico'.' [/home/maestro/.vscode/extensions/WinForms/winforms/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj]
    0 Warning(s)
    1 Error(s)

_(I found a workaround by renaming the appropriate files to the appropriate folder)_

--- a/src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj
+++ b/src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj
@@ -3,7 +3,7 @@
     https://github.com/dotnet/winforms/issues/2107#issuecomment-559289716
     https://github.com/microsoft/msbuild/issues/4923#issuecomment-554265394
     -->
-  <Import Project="Sdk.Props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   
   <PropertyGroup>
     <AssemblyName>Microsoft.VisualBasic.Forms</AssemblyName>

--- a/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj
+++ b/src/System.Windows.Forms.Design/src/System.Windows.Forms.Design.csproj
@@ -121,11 +121,11 @@
     <EmbeddedResource Include="Resources\System\WinForms\Design\default.ico">
       <LogicalName>System.Windows.Forms.Design.default</LogicalName>
     </EmbeddedResource>
-    <EmbeddedResource Include="Resources\System\WinForms\Design\professional1.ico">
-      <LogicalName>System.Windows.Forms.Design.professional1</LogicalName>
+    <EmbeddedResource Include="Resources\System\WinForms\Design\Professional1.ico">
+      <LogicalName>System.Windows.Forms.Design.Professional1</LogicalName>
     </EmbeddedResource>
-    <EmbeddedResource Include="Resources\System\WinForms\Design\professional2.ico">
-      <LogicalName>System.Windows.Forms.Design.professional2</LogicalName>
+    <EmbeddedResource Include="Resources\System\WinForms\Design\Professional2.ico">
+      <LogicalName>System.Windows.Forms.Design.Professional2</LogicalName>
     </EmbeddedResource>
     <EmbeddedResource Include="Resources\System\WinForms\Design\classic.ico">
       <LogicalName>System.Windows.Forms.Design.classic</LogicalName>
@@ -154,11 +154,11 @@
     <EmbeddedResource Include="Resources\System\WinForms\Design\Behavior\DesignerShortcutBox.ico">
       <LogicalName>System.Windows.Forms.Design.Behavior.DesignerShortcutBox</LogicalName>
     </EmbeddedResource>
-    <EmbeddedResource Include="Resources\System\WinForms\Design\Behavior\leftopen.ico">
-      <LogicalName>System.Windows.Forms.Design.Behavior.leftopen</LogicalName>
+    <EmbeddedResource Include="Resources\System\WinForms\Design\Behavior\leftOpen.ico">
+      <LogicalName>System.Windows.Forms.Design.Behavior.leftOpen</LogicalName>
     </EmbeddedResource>
-    <EmbeddedResource Include="Resources\System\WinForms\Design\Behavior\leftclose.ico">
-      <LogicalName>System.Windows.Forms.Design.Behavior.leftclose</LogicalName>
+    <EmbeddedResource Include="Resources\System\WinForms\Design\Behavior\leftClose.ico">
+      <LogicalName>System.Windows.Forms.Design.Behavior.leftClose</LogicalName>
     </EmbeddedResource>
     <EmbeddedResource Include="Resources\System\WinForms\Design\Behavior\rightopen.ico">
       <LogicalName>System.Windows.Forms.Design.Behavior.rightopen</LogicalName>

